block-streamer/src/control.rs

```diff
@@ -1,35 +1,46 @@
 use anyhow::Result;
+use local_storage::provider::SqliteStorageProvider;
+use local_storage::storage::Storage;
 use parity_scale_codec::Decode;
 use std::net::SocketAddr;
 use std::path::PathBuf;
+use std::rc::Rc;
```

Comment:
If you're in a tokio thread, you may want to use their [sleep](https://docs.rs/tokio/latest/tokio/time/fn.sleep.html) function.